jobsdb/jobsdb.go

```diff
@@ -534,15 +562,15 @@ var dbInvalidJsonErrors = map[string]struct{}{
 // instanceBackupFailedAndAborted = true => the individual jobdb backsup failed and aborted jobs only
 // pathPrefix = by default is the jobsdb table prefix, is the path appended before instanceID in s3 folder structure
 func (jd *Handle) registerBackUpSettings() {
```

Comment:
since this doesn't seem to contain only backup settings can we remove the function and move its contents to `loadConfig`?